iib/workers/tasks/build_merge_index_image.py

```diff
@@ -249,6 +281,11 @@ def handle_merge_request(
         )
 
         if deprecation_bundles:
+            if is_image_fbc(intermediate_image_name):
+                err_msg = 'File-Based catalog image type is not supported for deprecation yet.'
```

Comment:
Intermediate image would be FBC when source_image is FBC?

Reply:
yes